api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -336,6 +339,9 @@ message AccessListReviewMetadata {
 
   // RemovedMembers are the members that were removed as part of the review.
   repeated string RemovedMembers = 6 [(gogoproto.jsontag) = "removed_members,omitempty"];
+
```

Comment:
The `metadata.name` is the immutable identifier for access list. Should we include that too?

Reply:
the metadata.name is not needed for the changes in the follow up PRs after this one. I think it makes more sense to only add the field if it's intended to be used, if we end up needing it we can add it later.